docs/clients/lcd-rest-api.yaml

```diff
@@ -714,15 +714,15 @@ definitions:
   ValidatorAddress:
     type: string
     description: bech32 encoded addres
-    example: cosmosval:zgnkwr7eyyv643dllwfpdwensmgdtz89yu73zq
+    example: cosmosvaloper:zgnkwr7eyyv643dllwfpdwensmgdtz89yu73zq
   PubKey:
     type: string
     description: bech32 encoded public key
     example: cosmospub:zgnkwr7eyyv643dllwfpdwensmgdtz89yu73zq
   ValidatorPubKey:
     type: string
     description: bech32 encoded public key
-    example: cosmosvalpub:zgnkwr7eyyv643dllwfpdwensmgdtz89yu73zq
+    example: cosmosvaloperpub:zgnkwr7eyyv643dllwfpdwensmgdtz89yu73zq
```

Comment:
Wait, is this not the consensus pubkey? If not, why does Voyager need the operator's public key?

Reply:
Indeed!